account_wallet/models/account_wallet_type.py

```diff
@@ -0,0 +1,94 @@
+# © 2015  Laetitia Gangloff, Acsone SA/NV (http://www.acsone.eu)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountWalletType(models.Model):
+    _name = "account.wallet.type"
+    _description = "Wallet Type"
+    _check_company_auto = True
+
+    name = fields.Char(translate=True, required=True)
+    sequence_id = fields.Many2one(
+        comodel_name="ir.sequence",
+        string="Wallet Sequence",
+        copy=False,
+        check_company=True,
+        help="This field contains the information related to the numbering "
+        "of the wallet of this type.",
+        required=True,
+    )
+    account_id = fields.Many2one(
+        comodel_name="account.account",
+        string="Account",
+        ondelete="restrict",
+        index=True,
+        required=True,
+    )
+    journal_id = fields.Many2one(
+        comodel_name="account.journal",
+        string="Journal",
+        ondelete="restrict",
+        help="Journal use to empty the wallet",
+        required=True,
+    )
+    product_id = fields.Many2one(
+        comodel_name="product.product",
+        string="Product",
+        ondelete="restrict",
+        help="Product use to fill the wallet",
```

Comment:
```suggestion
        help="Product use to fill the wallet",
        domain="[('type', '=', 'service')]",
```

What about to add a domain (like wizard.account_move_credit_notes.wallet product_id field).

(just a question : but at the first sight, I think that we only requires "services" in such field.

Reply:
Done here : https://github.com/acsone/wallet/pull/3